evm/src/executor/inspector/tracer.rs

```diff
@@ -67,6 +69,29 @@ impl Tracer {
             trace.address = address;
         }
     }
+
+    pub fn start_step(&mut self, step: CallTraceStep) {
+        let trace = &mut self.traces.arena
+            [*self.trace_stack.last().expect("can't start step without starting a trace first")];
+
+        trace.trace.step_stack.push(trace.trace.steps.len());
+        trace.trace.steps.push(step);
+    }
+
+    pub fn fill_step(&mut self, gas: u64, status: Return) {
+        let trace = &mut self.traces.arena
+            [*self.trace_stack.last().expect("can't fill step without starting a trace first")];
+        let step_idx =
+            trace.trace.step_stack.pop().expect("can't fill step without starting a step first");
+        let step = &mut trace.trace.steps[step_idx];
+
+        step.gas_cost = step.gas - gas;
+
+        // Error codes only
+        if status as u8 > 0x50 {
```

Comment:
can we make this a constant,

or perhaps we could even do

`>= Return::OutofGas as u8`

https://github.com/bluealloy/revm/blob/5572e303c7963727538883de817f9b52ca304f50/crates/revm/src/instructions.rs#L50

Reply:
yeahhhhh i made it constant at first but reverted for some reason. Fixed!